aten/src/ATen/core/jit_type.h

```diff
@@ -1384,9 +1385,42 @@ struct ClassType;
 using ClassTypePtr = std::shared_ptr<ClassType>;
 using ::torch::jit::script::CompilationUnit;
 using ::torch::jit::Function;
+struct SerializableType;
+using SerializableTypePtr = std::shared_ptr<SerializableType>;
+
+struct CAFFE2_API SerializableType : public Type {
```

Comment:
"NamedType" might be more accurate. Afterall, all of our types are serializable. I assume this will be come the shared parent for Module, Interface, and NamedTuple types?